bin/rex

```diff
@@ -11,7 +11,14 @@ use warnings;
 
 # VERSION
 
-use Rex::CLI;
+use Rex::DetectCLI;
+my $cli = detect_cli;
+
+eval "require 'Rex/CLI/$cli.pm'";
```

Comment:
you're right. and there is also: https://github.com/RexOps/Rex/blob/master/lib/Rex/Require.pm which is doing the same.